src/org/ice1000/devkt/ui/annotator.kt

```diff
@@ -26,6 +26,18 @@ class KotlinAnnotator {
 			is KtAnnotationEntry -> {
 				document.highlight(element, colorScheme.annotations)
 			}
+			
+			is KtTypeProjection -> {
+				//<泛型>
+				if (element.prevSibling?.node?.elementType == KtTokens.LT
+						&& element.nextSibling?.node?.elementType == KtTokens.GT) {
+					document.highlight(element, colorScheme.numbers)
+				}
+			}
+
+			is KtFunction -> {
+				document.highlight(element.nameIdentifier ?: return, colorScheme.functionDeclaration)
```

Comment:
Are you sure `functionDeclaration` is defined? Do you have uncommited files?

Reply:
No, I am not sure QAQ